single-line-diagram-core/src/test/java/com/powsybl/sld/iidm/TestCase1.java

```diff
@@ -61,7 +61,7 @@ public void test() {
         new BlockOrganizer().organize(g);
 
         // calculate coordinates
-        LayoutParameters layoutParameters = new LayoutParameters()
+        layoutParameters = new LayoutParameters()
```

Comment:
Maybe not very important but it is probably a little bit weak as a design. Maybe just have an `abstract LayoutParameters getLayoutParameter()` in the base class?